.github/workflows/fpb-lint.yml

```diff
@@ -2,19 +2,109 @@ name: free-programming-books-lint
 
 on: [push, pull_request]
 
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_RUN_ID: ${{ github.run_id }}
+  PR: ${{ github.event.pull_request.html_url }}
+
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v3
+    - name: Use Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
```

Comment:
Not needed since uses `actions/github-script` to clean output

Reply:
oh right, I forgot to remove it, thanks